index.d.ts

```diff
@@ -54,6 +54,15 @@ declare namespace open {
 		readonly allowNonzeroExitCode?: boolean;
 	}
 
+	interface OpenAppOptions extends Omit<Options, 'app'> {
+		/**
+		Optional arguments to open.
```

Comment:
This description is not really correct.
```suggestion
		Arguments passed to the app.
```